src/Explorer/Notebook/NotebookComponent/epics.ts

```diff
@@ -944,6 +944,39 @@ const traceNotebookKernelEpic = (
   );
 };
 
+const resetCellStatusOnExecuteCanceledEpic = (
+  action$: Observable<actions.ExecuteCanceled>,
+  state$: StateObservable<AppState>
+): Observable<actions.UpdateCellStatus> => {
+  return action$.pipe(
+    ofType(actions.EXECUTE_CANCELED),
```

Comment:
>EXECUTE_CANCELED [](start = 19, length = 16)

your description says EXECUTE_FAILED? hopr this is the right one?

Reply:
Oops! Updated the description. The code is correct.